### PR TITLE
RD-3069 Update documentation for Blueprint flow changes

### PR DESCRIPTION
**Pull request by stepek**
Adde changes in config part of widgets and added new options to deploy widgets.

### CODE REVIEW

File: content/working_with/console/widgets/blueprints.md [hunk 1]
```diff
@@ -84,3 +98,7 @@ See Settings section for details on how to turn on/off this feature.
 * `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds
 * `Enable click to drill down` - This option enables redirecting to the blueprint’s drill-down page upon clicking on a specific blueprint. Default: True
 * `Display style` - Can be either Catalog or table. The deployments status column is only available in list mode.  Default: table
+* `Show Composer options` - This options anables to show composer options in menu and in blueprint list. Default: false
```
Comment (qooban) [inline content/working_with/console/widgets/blueprints.md:101]:
```suggestion
* `Show Composer options` - This options enables to show {{< param cfy_composer_name >}} options in menu and in the blueprints list. Default: false
```

File: content/working_with/console/widgets/blueprints.md [hunk 2]
```diff
@@ -33,9 +33,18 @@ There are also action buttons to upload a blueprint, create deployment, delete b
 
 #### Uploading a Blueprint
 
+##### Using marketplace
+
 1. Click the **Upload** button.
-2. In the Upload Blueprint dialog, provide the URL of the remote archive in which the blueprint is located or select a local blueprint archive.
-3. Enter the `Blueprint name` and `Blueprint YAML file`.   
+2. In the menu select **Upload from Marketplace** option.
+3. In the Blueprint marketplace dialog click **Upload** button on one of blueprint from the catalog.
```
Comment (qooban) [inline content/working_with/console/widgets/blueprints.md:40]:
```suggestion
2. In the menu, select **Upload from Marketplace** option.
3. In the Blueprint marketplace dialog, click **Upload** button on one of the blueprints shown in a table.
```
as we are showing examples in table view right now.

File: content/working_with/console/widgets/blueprints.md [hunk 3]
```diff
@@ -44,6 +53,11 @@ There are also action buttons to upload a blueprint, create deployment, delete b
 The default visibility is "Tenant", and according to the logged-in user's permissions you can also choose other levels of [resource visibilities]({{< relref "working_with/manager/resource-visibility.md" >}}).<br>
 6. Click **Upload**.
 
+##### Using composer
+
+1. Click the **Upload** button
```
Comment (qooban) [inline content/working_with/console/widgets/blueprints.md:58]:
```suggestion
1. Click the **Upload** button.
```

File: content/working_with/console/widgets/blueprints.md [hunk 4]
```diff
@@ -44,6 +53,11 @@ There are also action buttons to upload a blueprint, create deployment, delete b
 The default visibility is "Tenant", and according to the logged-in user's permissions you can also choose other levels of [resource visibilities]({{< relref "working_with/manager/resource-visibility.md" >}}).<br>
 6. Click **Upload**.
 
+##### Using composer
```
Comment (qooban) [inline content/working_with/console/widgets/blueprints.md:56]:
```suggestion
##### Using {{< param cfy_composer_name >}}
```

File: content/working_with/console/widgets/blueprints.md [hunk 5]
```diff
@@ -44,6 +53,11 @@ There are also action buttons to upload a blueprint, create deployment, delete b
...
+
+1. Click the **Upload** button
+2. In the menu click **Generate in the Composer** option.
+3. Creator will be opened in the new card and there you can create blueprint.
```
Comment (qooban) [inline content/working_with/console/widgets/blueprints.md:60]:
```suggestion
3. {{< param cfy_composer_name >}} will be opened in a new tab and there you can create blueprint.
```

File: content/working_with/console/widgets/blueprints.md [hunk 6]
```diff
@@ -84,3 +98,7 @@ See Settings section for details on how to turn on/off this feature.
 * `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds
 * `Enable click to drill down` - This option enables redirecting to the blueprint’s drill-down page upon clicking on a specific blueprint. Default: True
 * `Display style` - Can be either Catalog or table. The deployments status column is only available in list mode.  Default: table
```
Comment (qooban) [inline content/working_with/console/widgets/blueprints.md:100]:
```suggestion
* `Display style` - Defines how the blueprints list should be displayed. Can be either Catalog or Table. The deployments status column is only available in list mode.  Default: Table
```

File: content/working_with/console/widgets/blueprints.md [hunk 7]
```diff
@@ -84,3 +98,7 @@ See Settings section for details on how to turn on/off this feature.
...
+* `Show Composer options` - This options anables to show composer options in menu and in blueprint list. Default: false
+* `Marketplace tabs` - Allow to edit tabs in Marketplace modal.
+* `Marketplace display style` - Can be either Catalog or table of market place modal.  Default: table
```
Comment (qooban) [inline content/working_with/console/widgets/blueprints.md:103]:
```suggestion
* `Marketplace display style` - Defines how the Blueprints Marketplace should be displayed. Can be either Catalog or Table.  Default: Table
```

File: content/working_with/console/widgets/blueprints.md [hunk 8]
```diff
@@ -84,3 +98,7 @@ See Settings section for details on how to turn on/off this feature.
...
+* `List of fields to show in the marketplace table` - Allow to change visible columns in the marketplace only in table mode.
```
Comment (qooban) [inline content/working_with/console/widgets/blueprints.md:104]:
```suggestion
* `List of fields to show in the marketplace table` - Allow to change the list of visible columns in the Blueprint Marketplace. Works only when Marketplace display style is set to Table. Default: Name, Description.
```

**Review by qooban — COMMENTED** [review]
I did partial review. More comments may be added tomorrow.
I see no screenshots updates, I suspect they are to be updated as well.

**Comment by kubama** [comment]
I have nothing to add to @qooban's comments, so I'll opt out from the review.

**Comment by stepek** [comment]
Changed branch name, please check this commit https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/1960

**Pull request closed without merging**